gateway-protocol-impl/src/main/resources/gateway-service-config.json

```diff
@@ -0,0 +1,50 @@
+{
+  "methodConfig": [
+    {
+      "name": [
+        {"service": "gateway_protocol.Gateway", "method": "ActivateJobs"},
+        {"service": "gateway_protocol.Gateway", "method": "BroadcastSignal"},
+        {"service": "gateway_protocol.Gateway", "method": "CancelProcessInstance"},
+        {"service": "gateway_protocol.Gateway", "method": "CompleteJob"},
+        {"service": "gateway_protocol.Gateway", "method": "DeleteResource"},
+        {"service": "gateway_protocol.Gateway", "method": "EvaluateDecision"},
+        {"service": "gateway_protocol.Gateway", "method": "FailJob"},
+        {"service": "gateway_protocol.Gateway", "method": "ResolveIncident"},
+        {"service": "gateway_protocol.Gateway", "method": "SetVariables"},
+        {"service": "gateway_protocol.Gateway", "method": "StreamActivatedJobs"},
+        {"service": "gateway_protocol.Gateway", "method": "ThrowError"},
+        {"service": "gateway_protocol.Gateway", "method": "Topology"},
+        {"service": "gateway_protocol.Gateway", "method": "UpdateJobRetries"},
+        {"service": "gateway_protocol.Gateway", "method": "UpdateJobTimeout"}
+      ],
+      "waitForReady": true,
+      "retryPolicy": {
+        "maxAttempts": 5.0,
+        "initialBackoff": "0.1s",
+        "maxBackoff": "5s",
+        "backoffMultiplier": 3.0,
```

Comment:
❓ In the Java and Go client we use 1.6 as the backoff factor. Why did you choose 3 here?

Reply:
The scaling was very low imo, esp. with a max attempts of 5.

Starting with 0.1 and 1.6, you get 0.1, 0.16, 0.256, 0.409, 0.655.
Starting with 0.1 and 3.0, you get 0.1, 0.3, 0.6, 0.9, 2.7.

But to be honest, feel free to put whatever in there. I didn't spend that much time on it.

Reply:
That's a good point. Introducing this config is a good moment to improve the default.